### PR TITLE
Skip add metadata for SPIR target

### PR DESCRIPTION
**Pull request by Solaryee**
This is a sub PR to add SPIR target in openxla.
There is no need to add range metadata for SPIR target, so just skip it in `AddRangeMetadata` function.

### CODE REVIEW

**Comment by Zantares** [comment]
Sub PR of https://github.com/openxla/xla/pull/9042

**Review by penpornk — APPROVED** [review]
Thank you for the PR!

File: xla/service/llvm_ir/llvm_util.cc [hunk 1]
```diff
@@ -50,6 +50,7 @@ limitations under the License.
 #include "llvm/Support/Casting.h"
 #include "llvm/Support/CodeGen.h"
 #include "llvm/Support/raw_ostream.h"
+#include "llvm/TargetParser/Triple.h"
```
Comment (akuegel) [inline xla/service/llvm_ir/llvm_util.cc:53]: This needs a corresponding dependency in the BUILD file for the llvm_util target.

Comment (ShengYang1) [reply xla/service/llvm_ir/llvm_util.cc:53]: Done.

**Review by akuegel — APPROVED** [review]

**Pull request closed without merging**